pkg/golinters/gci.go

```diff
@@ -43,13 +47,17 @@ func NewGci() *goanalysis.Linter {
 			var issues []goanalysis.Issue
 
 			for _, f := range fileNames {
-				diff, err := gci.Run(f, &gci.FlagSet{LocalFlag: localFlag})
+				source, result, err := gci.Run(f, &gci.FlagSet{LocalFlag: localFlag})
 				if err != nil {
 					return nil, err
 				}
-				if diff == nil {
+				if bytes.Equal(source, result) {
```

Comment:
It's not necessary, since you are doing it here: https://github.com/daixiang0/gci/pull/9/files#diff-1554c40a9d7ee0dfba770905afabc360R362

Reply:
```suggestion
				if source == nil || result == nil {
```